src/utils.js

```diff
@@ -162,18 +166,16 @@ export function saveUser(userId) {
                 );
               }
             }).catch((err) => logger.error(`User ${realname} is already in the database`, err));
+          }
         });
     }).catch(
       () => logger.error(`Trying to save bot or disabled user ${userId}`)
     );
 }
 
 export async function userExists(userId) {
-  return database
-    .get(
-      'SELECT * FROM users WHERE user_id=$userId',
-      {$userId: userId}
-    ).then((result) => !!result);
+  const records = (await listRecords());
```

Comment:
No need to have parentheses around this expression. Also don't you want to filter users directly on airtable?